nupic/algorithms/anomaly.py

```diff
@@ -164,6 +166,17 @@ def __str__(self):
     return "Anomaly:\tmode=%s\twindowSize=%r" % (self._mode, windowSize)
 
 
+  def __cmp__(self, other):
```

Comment:
What's the usage for this? I can imagine a use case for `__eq__` but `__cmp__` doesn't seem right.